backend/authentication/migrations/0002_auto_20190221_0228.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.1.7 on 2019-02-21 02:28
+
```

Comment:
This should also be deleted.

Reply:
We migrated which we believe should have deleted this.